docs/DEV.md

```diff
@@ -0,0 +1,48 @@
+# Hush Line Dev
+
+| Contents |
+|-|
+| 1. [Local Development](#local-development) |
+
+## Local Development
+
+This guide preps your machine to run Hush Line locally using the included `Makefile`.
+
+### Mac
+
+| Tested Platform | Date |
+|-|-|
+| Mac M2 | Apr. 2024 |
+
+#### Install Packages
+- `/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"`
+- `brew install python git git-lfs nginx redis rust`
+- `echo 'export PATH="/opt/homebrew/bin:$PATH"' >> ~/.zshrc`
+- `curl -sSL https://install.python-poetry.org | python3 -`
+- `echo 'export PATH="$HOME/.local/bin:$PATH"' >> ~/.zshrc`
+- `source ~/.zshrc`
+
+#### Clone the Repo
+- `https://github.com/scidsg/hushline.git`
+- `cd hushline`
+- `git switch dev-env` # Temporary until we merge back into Main
+- `python -m venv venv`
+- `source venv/bin/activate`
+- `poetry install`
+- `source env.sh`
```

Comment:
These last lines should just be `poetry run make init-db run`. We shouldn't run the migrations in dev as part of the main workflow.

Reply:
✅

Reply:
✅